int32 allied_type
---
float32 exposure_time
int32 error